tmva/tmva/inc/TMVA/DNN/Minimizers.h

```diff
@@ -58,8 +57,7 @@ class TGradientDescent
 
 private:
    size_t   fBatchSize; ///< Batch size to use for the training.
-   size_t   fStepCount; ///< Number of steps performed in the current
-   ///< training sessiong.
+   size_t fStepCount;   ///< Number of steps performed in the current training session
```

Comment:
clang-format complains otherwise. My change is actually a typo fix `sessiong -> session`.